diff --git a/code/xprize/prescribe.py b/code/xprize/prescribe.py
index e446913678..8978da29d5 100644
--- a/code/xprize/prescribe.py
+++ b/code/xprize/prescribe.py
@@ -9,7 +9,8 @@ import time
 import json
 import re

-sys.path.append(os.path.expanduser("logger"))
+sys.path.append(os.path.expanduser("~/coronasurveys/code/xprize/logger"))
+print("adding path ",os.path.expanduser("~/coronasurveys/code/xprize/logger"))
 import coronasurveys_utils

 ##########################################################################################
@@ -23,7 +24,7 @@ subprocess_timeout = 19800 # 5.5 hours in seconds
 #os.environ["PATH"] += os.pathsep + opt_conda_path + os.pathsep + bin_path

 # workdir = "~/work"
-workdir="."
+workdir="~/coronasurveys/code/xprize"

 def zeroOutput(start_date_str: str,
                end_date_str: str,